test/embed.test.js

```diff
@@ -161,28 +161,28 @@ describe('Embed Tests', () => {
 
   it('Sanitizes Malicious URLs', async () => {
     // eslint-disable-next-line no-script-url
-    const { headers, body } = await embed('javascript:alert(1)');
+    const { headers, body } = await embed('javascript:alert(1)', { kind: 'embed-' });
```

Comment:
why this kind?

Reply:
I agree, we probably shouldn't add any classes in case of illegal URLs.

Reply:
Oh right; doiiiii great catch